tests/js/advancedCounter.scrypttest.js

```diff
@@ -3,13 +3,12 @@ const { bsv, buildContractClass, getPreimage, toHex, num2bin, SigHashPreimage, R
 const {
   inputIndex,
   inputSatoshis,
-  tx,
+  newTx,
   DataLen,
   compileContract
 } = require('../../helper');
 
-// make a copy since it will be mutated
-const tx_ = bsv.Transaction.shallowCopy(tx)
+
```

Comment:
Let's put it here since it's global in each scrypttest. U may have multiple `describe`s.
`const tx = newTx()`

Reply:
9a531461eca019f0a14ad12158cfe4e098d0e462